### ISSUE TITLE
API

### ISSUE THREAD

**Issue by iltempe** [timwis/jkan#82]
HI! I've implemented a simple reading API to have access via http to all datasets and metadata.

You can look at this at http://iltempe.github.io/opendatagentediprato/ going to link http://iltempe.github.io/opendatagentediprato/api/datasets.json

regards!

**Comment by timwis** [comment]
Hey @iltempe - cool idea! That was the sentiment behind #34, which uses the data.json specification. You can see your at http://iltempe.github.io/opendatagentediprato/data.json

**Comment by iltempe** [comment]
Yes! i've seen. My idea is to develop the API folder to create a complete API! have you some suggestions?

**Comment by timwis** [comment]
Oh, I see. The only problem with that is that you can't generate an individual json file for each dataset because we're already generating an individual html file for each one, and you can only generate one file per collection item. We discussed that a bit in #72, but @Floppy has an interesting idea that may work out for it.

**Comment by Floppy** [comment]
I'm also currently proposing and working on an enhancement to Jekyll to allow >1 output file per item, in multiple formats, that would enable exactly this: https://github.com/jekyll/jekyll/issues/3041 (WIP PR in https://github.com/jekyll/jekyll/pull/4742)

**Comment by iltempe** [comment]
Nice! at the moment with solution like mine I think is possible create a single JSON file also making some query on datasets, for example to see all datatets of certain categories.

**Comment by timwis** [comment]
Oh, awesome @Floppy! I ruled that possibility out after reading jekyll/jekyll#1305. Really looking forward to your PR.

**Comment by loleg** [comment]
I'm a bit confused: is there actually an unmerged PR linked to this discussion?

**Comment by Floppy** [comment]
No, and I never managed to get round to doing the Jekyll work I was doing that might have helped make this happen, unfortunately. Maybe one day when I retire 😆
